step-automation-packages/step-automation-packages-manager/src/main/java/step/automation/packages/AutomationPackageManager.java

```diff
@@ -504,20 +518,24 @@ protected void persistStagedEntities(Staging staging,
             objectEnricher.accept(execTasksParameter.getExecutionsParameters());
         }
 
-        if (!automationPackageHookRegistry.onCreate(AutomationPackageFragmentYaml.SCHEDULES_FIELD_NAME, staging.taskParameters, objectEnricher)) {
-            // by default just save task parameters via accessor
-            for (ExecutiontTaskParameters taskParameter : staging.taskParameters) {
-                executionTaskAccessor.save(taskParameter);
+        // save task parameters and additional objects via hooks
+        List<HookEntry> hookEntries = new ArrayList<>();
+        hookEntries.add(new HookEntry(AutomationPackageFragmentYaml.SCHEDULES_FIELD_NAME, staging.taskParameters));
```

Comment:
@iegorov777 I still don't understand why we have a special case for schedules here and why we need a DefaultExecutionTaskParameterHook in AutomationPackageManager. For me, that could be completely moved to the AutomationPackageSchedulerHook. If the hook from the schedule plugin doesn't exists we should just ignore the schedules as we do for alerting rules, or did I miss something?

Reply:
@david-stephan 
If we may just ignore the `schedules` section in yaml in case the `AutomationPackageSchedulerHook` is not registered (in `SchedulerPlugin`), than we really can remove the default hook and this makes everything much easer.

But I tried to keep the current behaviour and persist schedules via `executionTaskAccessor.save(entity)` even if the scheduler hook is not registered. 

So can we just add the `AutomationPackageSchedulerHook` in ``SchedulerPlugin` and remove all `default` logic from AM Manager, what do you think?